manifests/Cromha Expansion.yaml

```
@@ -0,0 +1,14 @@
+name: Cromha Expansion Plugin
+authors: @OcelotWalrus/Cromha, @Lumbar527
```

```suggestion
authors: "@OcelotWalrus/Cromha, @Lumbar527"
```
Needs some form of quotes to not break YAML syntax

---

Done